CONTRIBUTING.md

```diff
@@ -23,3 +23,34 @@ use GitHub pull requests for this purpose. Consult [GitHub Help] for more
 information on using pull requests.
 
 [GitHub Help]: https://help.github.com/articles/about-pull-requests/
+
+## Building Dagger
+
+Dagger is built with [`bazel`](https://bazel.build).
+
+### Building Dagger from the command line
+
+* Go to the [Bazel installation page](https://docs.bazel.build/versions/master/install.html) and install Bazel.
+  * Learn more about Bazel targets [here](https://docs.bazel.build/versions/master/build-ref.html).
+* Clone the Dagger repository: `git clone git@github.com:google/dagger.git dagger`
+* Build the Dagger project with `bazel build`
+  * If you get the following error: `ERROR: missing input file '@androidsdk//:build-tools/26.0.2/aapt'`, install the missing build tools version with sdkmanager.
+* You can install the Dagger libraries in your **local maven repository** by running the `./util/install-local-snapshot.sh` script.
+  * It will build the libraries and install them with a `LOCAL-SNAPSHOT` version.
+* You can **run all tests** with `bazel test //...` and a single test with the path and class name, e.g. `bazel test ///javatests/dagger/internal/codegen:KeyFactoryTest`.
+
+### Importing the Dagger project in Android Studio
+
+* In *Android Studio*, go to `Preferences > Plugins`.
+  * search for `bazel` and install the plugin.
+  * If no result shows up, click on `Search in repositories`, search for `bazel` and install the plugin.
+  * The Bazel plugin is available for Android Studio 3.2 or greater.
+* Select `Import Bazel Project`.
+* Input the path to the Dagger project under `workspace`, click `Next`.
+* Select `Generate from BUILD file`, type `BUILD` in the `Build file` input, click `Next`.
+* In the `Project View` form, uncomment one of the `android_sdk_platform` lines. Pick one that you have installed, then click `Finish`.
```

Comment:
I don't see this as an option. Just `android`

![image](https://user-images.githubusercontent.com/1361086/50554193-167f8280-0c6b-11e9-9ea0-5c985164c4df.png)

Reply:
IntelliJ or AndroidStudio?